lib/src/components/NxToast/NxToastContainer.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useRef, useEffect } from 'react';
+import NxToastContainerContext from './contexts';
+import {reject, last} from 'ramda';
+
+import { NxToastContainerProps, nxToastContainerPropTypes } from './types';
+export { NxToastContainerProps };
+
+import './NxToast.scss';
+
+const NxToastContainer = (props: NxToastContainerProps) => {
+  const { children } = props;
+
+  const ref = useRef<HTMLDivElement | null>(null);
+
+  // When all toasts are closed, the focus should return to the last element that received focus that was not
+  //a toast element.
+  const activeElementNotToast = useRef<HTMLElement | null>(null);
+
+  const onToastClosing = (toast:HTMLElement | null) => {
+    const closeBtns = Array.from(ref.current?.querySelectorAll<HTMLButtonElement>('.nx-toast .nx-btn--close') ?? []);
```

Comment:
You'll need to adjust the formatting, but I believe this will fix the issue when focus getting lost when you close too many toasts too fast with the keyboard
```suggestion
    const closeBtns = Array.from(ref.current?.querySelectorAll<HTMLButtonElement>('.nx-toast:not(.nx-toast--closing) .nx-btn--close') ?? []);
```

Reply:
> You'll need to adjust the formatting, but I believe this will fix the issue with focus getting lost when you close too many toasts too fast with the keyboard

Added and formatted here: [c339934](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/c3399348327fac99c62c98d6a7a344448078ab3c)